extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -2159,6 +2159,63 @@ describe('amp-a4a', () => {
     });
   });
 
+  describe('refresh', () => {
+    let sandbox;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should effectively reset the slot and invoke given callback', () => {
```

Comment:
Can we add a couple more tests:

- Verify that if ad promise does not fill, we do nothing (no placeholder, no destroy frame, etc)
- Verify that unlayoutCallback called during refresh works correctly
- Verify that unlayoutCallback followed by resume works correctly
- Verify refresh does not increment ifi as part of the request and include refresh count